sqlwhat/sct_context.py

```diff
@@ -0,0 +1,9 @@
+from protowhat.sct_context import create_sct_context, get_checks_dict
+from sqlwhat import checks
+
+SCT_DICT = get_checks_dict(checks)
```

Comment:
Why the all capital variable name?

Reply:
Also let's add back the old comment?
'used in Chain and F, to know what methods are available'

Reply:
These are constants generated once when loading the module.
They aren't used by `test_exercise` anymore (we use the protowhat functions there), but can be used when `test_exercise` isn't used.

Reply:
Can you give an example on usage without using test_exercise?

Reply:
Jeroen: In a repl, is already used for sct auto complete hints in teach